jennchenn/uwexplore
# filtering menu, disabled unused filters

## Implementation description

- Can now filter by course code
- Also, disabled the add button on search results that can't be added to the calendar



## Steps to test

1. Search for something
2. Apply course code filter
3. Verify only courses with the applied filter appear 



## What should reviewers focus on?

- Filters are removed if the filtering menu is closed... not ideal for long term but good enough for now?
- OR, I can have them still be applied, but not reflect as chosen/checked off in the filtering menu if the user closes/opens it again
